archive/untar.ts

```diff
@@ -222,7 +223,7 @@ export class Untar {
         // Ignore checksum header
         continue;
       }
-      sum += header[i];
+      sum += header[i]!;
```

Comment:
Since `i` gets no larger than HEADER_LENGTH and header is a Uint8Array with the size of HEADER_LENGTH this should never be out of bounds